skills/unity-vrc-udon-sharp/references/dynamics.md

```
@@ -28,29 +28,53 @@ Contacts provide a collision detection system between **Senders** and **Receiver
 #### Contact Sender
 
 1. Add `VRC Contact Sender` component to a GameObject
-2. Configure `Radius` (collision size)
-3. Set `Content Type` (identifies what kind of contact this is)
+2. Choose `Shape Type`: `Sphere` or `Capsule`
+3. Configure `Radius` (and `Height` for Capsule)
+4. Set `Content Type` (identifies what kind of contact this is)
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)